app/styles/style-flat.sass

```diff
@@ -167,20 +176,20 @@ $forest: #20572B
     border: 1px solid $navy
     color: $navy
     
-  .btn-forest
+  .btn-secondary, .btn-forest
     background-color: $forest
     color: white
     
-  .btn-forest-alt
+  .btn-secondary-alt, .btn-forest-alt
     background-color: white
     border: 1px solid $forest
     color: $forest
     
-  .btn-gold
+  .btn-tertiary, .btn-gold
     background-color: $gold
     color: white
     
-  .btn-gold-alt
+  .btn-tertiary-alt, .btn-gold-alt
```

Comment:
Mmm... Yeah let's get rid of both of those. I'm tempted to keep btn-primary in so it overwrites the bootstrap color, but I'm not actually sure which color should be considered primary.